packages/server/tests/acceptance.spec.ts

```diff
@@ -360,14 +372,50 @@ describe('RPC Server Integration Tests', async function () {
         expect(res.data.result).to.be.equal(false);
     });
 
-    it('should execute eth_sendRawTransaction legacy', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', ['0x' + legacyTransactionHex]);
-        expect(res.data.result).to.be.equal('0x9ffbd69c44cf643ed8d1e756b505e545e3b5dd3a6b5ef9da1d8eca6679706594');
-    });
-
-    it('should execute "eth_sendRawTransaction" london', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', ['0x' + londonTransactionHex]);
-        expect(res.data.result).to.be.equal('0xcdbbfb6400aab319f97d32c38e285f0d0399c2b48b683f04878b5f07eb0d50e3');
+    it('should execute "eth_sendRawTransaction" for legacy transactions', async function () {
+        const signedTx = await utils.signRawTransaction({
```

Comment:
nit: move object out to `defaultLegacyTransactionData`. Then you override `to` here